pkgs/applications/audio/DelayArchitect/default.nix

```
@@ -0,0 +1,40 @@
+{ lib, stdenv, fetchFromGitHub, libGL, libX11, libXext, libXrandr, libXinerama, libXcursor, freetype, alsa-lib, cmake, pkg-config, gcc-unwrapped }:
+
+stdenv.mkDerivation rec {
+  pname = "DelayArchitect";
```

```suggestion
  pname = "delayarchitect";
```
[Can't have uppercase letters](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming).